### ISSUE TITLE
g2o_viewer

### ISSUE THREAD

**Issue by sreeharshaparuchur1** [Shubodh/MR-project1-pgo#7]
Hey, 
I'm using Ubuntu 20.04 and on the first installation of g2o everything worked fine. When I tried to run g2o_viewer intel.g2o on the command line after changing the path it read "g2o_viewer: command not found". After a couple of days I decided to reinstall all the dependencies but while running cmake I faced an error. The error had been raised previously by another student and on deactivating all virtual environments the installation took place successfully. I then activated my environment and tried to run g2o_viewer in the dataset directory but faced the same error. I deactivated the environment but faced the same issue. My bash script modification seems to be alright as doing source ~/.bashrc doesn't throw up any errors but the issue persists.

**Comment by Shubodh** [comment]
Hi,
Please provide more details, the post is currently vague.
Go through [this issue](https://github.com/Shubodh/MR-project1-pgo/issues/5). What are the things you have tried mentioned there? Provide outputs that I have asked there (for example `ccmake`). I will not be able to help without more details.

**Comment by sreeharshaparuchur1** [comment]
I understand that the post may be vague but there was no problem after installation (I had faced a problem in the same the second time I tried to install, I checked out the very issue that you are directing me to and that helped me finish my installation). Simply running the command in the dataset directory didn't work. My teammate who also uses Ubuntu 20.04 is also facing a similar problem. 
https://pastebin.com/q5VFiNem
Are the outputs on running cmake and ccmake respectively. 
My make works completely. All the way to 100% the messages are short as this was already successfully done.
